__tests__/convention-test.ts

```diff
@@ -23,12 +29,20 @@ describe('Members tests', () => {
 		expect(utils.diagnosticsOnLine(4, camelCaseDiagnostics).length).toBe(1)
 	})
 
-
 	test('More than 25 characters', () => {
 		expect(utils.diagnosticsOnLine(14, lengthDiagnostics).length).toBe(1);
 	})
 
 	test('Starts with v', () => {
 		expect(utils.diagnosticsOnLine(23, vDiagnostics).length).toBe(1);
-	})
-})
+	});
+
+	test('property was not called \'dummy\'', () => {
+		expect(utils.diagnosticsOnLine(2, withoutDummyDiagnostics).length).toBe(0);
```

Comment:
No need to limit this test to a specific line. Check all the PropertyIsDummy diagnostics for the document to be 0.
```suggestion
		expect(withoutDummyDiagnostics.length).toBe(0);
```